packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -267,3 +267,29 @@ class WalletCreditsMessage(RabbitMessageBase):
 
     def routing_key(self) -> str | None:
         return f"{self.wallet_id}"
+
+
+class CreditsLimit(IntEnum):
+    ZERO = 0
+
+
+class WalletCreditsLimitReachedMessage(RabbitMessageBase):
+    channel_name: Literal["io.simcore.service.wallets-credit-limit-reached"] = Field(
+        default="io.simcore.service.wallets-credit-limit-reached", const=True
+    )
+    created_at: datetime.datetime = Field(
+        default_factory=lambda: arrow.utcnow().datetime,
+        description="message creation datetime",
+    )
+    service_run_id: str = Field(
+        ..., description="uniquely identitifies the service run"
+    )
+    user_id: UserID
+    project_id: ProjectID
+    node_id: NodeID
+    wallet_id: WalletID
+    credits: Decimal
+    credits_limit: CreditsLimit
+
+    def routing_key(self) -> str | None:
```

Comment:
Why returning None?

Reply:
Yes, I guess it is not necessary here, I will double-check the code.